### PR TITLE
Make sure that any of the future versions can load and execute older models.

### PR DESCRIPTION
**Pull request by VitalyFedyunin**
Helps to test #18952

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -11221,6 +11221,39 @@ def find_class(self, module, name):
...
+                  _1 = torch.zeros([10], dtype=6, layout=0, device=torch.device("cpu"))
+                  result = torch.to(torch.fill_(_1, 5), dtype=6, layout=0, device=torch.device("cpu"), non_blocking=False, copy=False)
+                  result2 = torch.rand([10], dtype=6, layout=0, device=torch.device("cpu"))
+                  result3 = torch.rand_like(result2, dtype=6, layout=0, device=torch.device("cpu"))
+                  _2 = torch.add(torch.add(result, result2, alpha=1), result3, alpha=1)
+                  return _2
+                ''',
+            'archive/attributes.pkl': b'\x80\x02](e.',
+            'archive/libs.py': b'op_version_set = 0\n',
+            'archive/model.json': b'{"protoVersion":"2","mainModule":{"torchscriptArena":{"key":"code/archive.py"},"name":"archive","optimize":true},"producerName":"pytorch","producerVersion":"1.0","libs":{"torchscriptArena":{"key":"libs.py"}}}'}
```
Comment (ezyang) [inline test/test_jit.py]: Line length too long? :o)

File: test/test_jit.py [hunk 2]
```diff
@@ -11221,6 +11221,39 @@ def find_class(self, module, name):
...
+        with TemporaryFileName() as fname:
+            archive_name = os.path.basename(os.path.normpath(fname))
+            print(archive_name)
```
Comment (ezyang) [inline test/test_jit.py]: probably don't want to commit this

**Comment by ezyang** [comment]
This looks good to me. Would be best to have someone on JIT look at it before landing.

**Comment by VitalyFedyunin** [comment]
@dzhulgakov hi! can you please stamp it. Going use BC tests a lot while working on #19092 and pin_memory thingy.

**Comment by ezyang** [comment]
Get someone like Zach or Michael to take a look at this PR, they will be
better for review.

Excerpts from Vitaly Fedyunin's message of 2019-04-12 19:58:44 +0000:
> @dzhulgakov hi! can you please stamp it. Going use BC tests a lot while working on #19092 and pin_memory thingy.
>

**Comment by VitalyFedyunin** [comment]
@zdevito @suo guys please take a look

**Review by ezyang — APPROVED** [review]

**Comment by SsnL** [comment]
this broke lint

**Comment by VitalyFedyunin** [comment]
> this broke lint

cleaning up

**Comment by VitalyFedyunin** [comment]
@ssnl #19313

**Pull request closed without merging**